### DISCUSSION TITLE
Compatibility Issue: Apollo Theme with PrimeReact on Next.js 14.2 - MultiSelect Checkbox Styling Issue

### DISCUSSION THREAD

**Discussion by Brunoiooo** [primefaces/community#2929] [PrimeReact Templates]
I would like to report a compatibility issue between the Apollo theme version for Next.js and the PrimeReact library. After updating the Next.js framework to version 14.2, I encountered visual issues with the PrimeReact MultiSelect component—checkboxes are not correctly hidden or styled, resulting in a visible standalone frame with an unstyled checkbox above it.

While the component does correctly select options, the unstyled checkbox issue affects the aesthetics and consistency of the user interface. My analysis suggests that this may be due to differences in the way checkboxes are generated across Next.js versions.

Is there any possibility of implementing adjustments in the Apollo theme to ensure full compatibility with the newer version of Next.js? This would allow us to take advantage of the latest framework features while maintaining a polished component appearance.

![image](https://github.com/user-attachments/assets/208aeb93-d4a7-4d81-8714-d7bcd1ab514f)

**Comment by soydanielgrosso** [comment]
I think this issue occurs due to updating to latests version of primerreact (for example: 10.8.4)

**Reply by Brunoiooo** [reply]
So, it seems like it's broken and not updated to current version of the PrimeReact.

**Reply by soydanielgrosso** [reply]
I think so... By the way, updating to nexts latest version also breakes everything when running the app

**Reply by Brunoiooo** [reply]
Yes, at first i though like u, it's depend of primeReact version... but i was wrong, it's bc of the next js framework version. For sure i see this different in way how primereact generating checkboxes. But also when i try to change primereact version... it doesn't matter... Matter is next js version...

PSSS... I hope someone from support of PrimeReact/Theme apollo see this discusion...

**Comment by Duduzera1997** [comment]
If it helps at all, the problem started in `primereact` v10.5.1.

**Comment by chrishj59** [comment]
Hi, have a look at [repo ](https://github.com/chrishj59/NextJsTemplate) - I have adapted the styles - it seems to work with NextJs 15.4.   PrimeReact 10.9.7 tailwindcss 4.1.1

**Reply by chrishj59** [reply]
@cagataycivici Please remove if this should not be here.

**Reply by bloodykheeng** [reply]
hei @chrishj59 add me to the repo
